config/default/user.role.preparer.yml

```diff
@@ -0,0 +1,85 @@
+uuid: 1d441765-e453-4e47-955e-6ee1f61756bd
```

Comment:
@cienvaras I'm unclear if this file should be ignored. I see we are ignoring this pattern in general. Since I'm installing this role within the module, this file does not do anything. But I'm still wondering if we should include it in the sync folder.